test/e2e.sh

```diff
@@ -71,6 +71,8 @@ $BIN deploy eck
 
 $BIN deploy opa policies test/opa/policies -v
 
+$BIN deploy gitops
```

Comment:
This should not be necessary, as `deploy all` should also deploy gitops controllers